pkg/service/v1alpha1/node.go

```diff
@@ -139,6 +139,12 @@ func (ns *node) NodeStageVolume(
 			logrus.Errorf("NodeStageVolume: failed to attachDisk for volume %v, err: %v", volumeID, err)
 			return nil, status.Error(codes.Internal, err.Error())
 		}
+		// If the access type is block, do nothing for stage
+		switch req.GetVolumeCapability().GetAccessType().(type) {
+		case *csi.VolumeCapability_Block:
+			return &csi.NodeStageVolumeResponse{}, nil
```

Comment:
Why are we not staging in case of raw block devices.

Reply:
NodeStage is not required in case of block volume request, we have to just return in case of block volume.